tools/javadoc-style/src/main/java/io/spine/tools/javadoc/style/formatting/JavadocStyler.java

```diff
@@ -43,29 +47,50 @@
 import static java.nio.file.Files.newBufferedWriter;
 
 /**
- * A formatter for Javadocs.
- *
- * <p>The formatter executes {@linkplain FormattingAction formatting actions}
- * for the Javadoc lines in a source file.
+ * Improves the style of Javadoc code by applying the passed formatting actions.
  */
-final class JavadocFormatter {
+public final class JavadocStyler {
 
+    private static final FluentLogger logger = FluentLogger.forEnclosingClass();
     private static final String TEMP_FILE_NAME = "temp_file_for_formatting.java";
 
     /** Formatting actions to perform. */
-    private final ImmutableList<FormattingAction> actions;
+    private final ImmutableList<Formatting> actions;
 
     /**
      * Creates an instance with the passed formatting actions.
      */
-    JavadocFormatter(FormattingAction... actions) {
+    @VisibleForTesting
+    JavadocStyler(Formatting... actions) {
         this.actions = ImmutableList.copyOf(actions);
     }
 
+    /**
+     * Improves the style for all Java files in the specified directory, including sub-directories.
+     */
+    public static void applyFormattingAt(Path directory) {
+        if (!Files.exists(directory)) {
```

Comment:
`Files.exists(path)` has its first parameter annotated as `@NonNull`. So, let's check it for `null` before passing on.